bedrock/firefox/views.py

```diff
@@ -414,16 +417,18 @@ def redirect_to(self):
 
         - Up-to-date Firefox users pass through.
         - Other Firefox users go to the new page.
-        - Non Firefox users go to the new page.
+        - Non Firefox users go to the configured page.
         """
         query = self.request.META.get('QUERY_STRING')
         query = '?' + query if query else ''
 
         user_agent = self.request.META.get('HTTP_USER_AGENT', '')
         if 'Firefox' not in user_agent:
-            return reverse('firefox.new') + query
+            return reverse(self.non_fx_redirect) + query
             # TODO : Where to redirect bug 757206
 
+        return None
```

Comment:
It was implicit, but agibson had made his explicit, then the Zen crept into my head, and this is much more obvious... to each their own I guess. :guardsman: